packages/katello/katello/katello.spec

```diff
@@ -174,6 +175,7 @@ Requires: findutils
 Requires: rh-mongodb34
 Requires: foreman-installer-%{name}
 Requires: rubygem-highline
+Requires: rh-mongodb34-syspaths
```

Comment:
Same applies here. I'd at least move this next to the `rh-mongodb34` line.

Reply:
I agree it looks like it pulls in what we need:

~~~
Dependencies Resolved

============================================================================================================================================================================================================
 Package                                                            Arch                                 Version                                         Repository                                    Size
============================================================================================================================================================================================================
Installing:
 rh-mongodb34-syspaths                                              x86_64                               3.0-14.el7                                      centos-sclo-rh                               4.3 k
Installing for dependencies:
 audit-libs-python                                                  x86_64                               2.8.1-3.el7                                     base                                          75 k
 checkpolicy                                                        x86_64                               2.5-6.el7                                       base                                         294 k
 libcgroup                                                          x86_64                               0.41-15.el7                                     base                                          65 k
 libpcap                                                            x86_64                               14:1.5.3-11.el7                                 base                                         138 k
 libsemanage-python                                                 x86_64                               2.5-11.el7                                      base                                         112 k
 policycoreutils-python                                             x86_64                               2.5-22.el7                                      base                                         454 k
 python-IPy                                                         noarch                               0.75-6.el7                                      base                                          32 k
 rh-mongodb34-boost-chrono                                          x86_64                               1.60.0-2.el7                                    centos-sclo-rh                                50 k
 rh-mongodb34-boost-filesystem                                      x86_64                               1.60.0-2.el7                                    centos-sclo-rh                                74 k
 rh-mongodb34-boost-iostreams                                       x86_64                               1.60.0-2.el7                                    centos-sclo-rh                                65 k
 rh-mongodb34-boost-program-options                                 x86_64                               1.60.0-2.el7                                    centos-sclo-rh                               157 k
 rh-mongodb34-boost-regex                                           x86_64                               1.60.0-2.el7                                    centos-sclo-rh                               299 k
 rh-mongodb34-boost-system                                          x86_64                               1.60.0-2.el7                                    centos-sclo-rh                                46 k
 rh-mongodb34-boost-thread                                          x86_64                               1.60.0-2.el7                                    centos-sclo-rh                                78 k
 rh-mongodb34-gperftools-libs                                       x86_64                               2.5-4.el7                                       centos-sclo-rh                               274 k
 rh-mongodb34-libstemmer                                            x86_64                               0-5.585svn.el7                                  centos-sclo-rh                                67 k
 rh-mongodb34-libunwind                                             x86_64                               1.2-3.el7                                       centos-sclo-rh                                64 k
 rh-mongodb34-mongo-tools                                           x86_64                               3.4.7-1.el7                                     centos-sclo-rh                                21 M
 rh-mongodb34-mongo-tools-syspaths                                  x86_64                               3.4.7-1.el7                                     centos-sclo-rh                               8.5 k
 rh-mongodb34-mongodb                                               x86_64                               3.4.9-1.el7                                     centos-sclo-rh                                22 M
 rh-mongodb34-mongodb-server                                        x86_64                               3.4.9-1.el7                                     centos-sclo-rh                                20 M
 rh-mongodb34-mongodb-server-syspaths                               x86_64                               3.4.9-1.el7                                     centos-sclo-rh                                22 k
 rh-mongodb34-mongodb-syspaths                                      x86_64                               3.4.9-1.el7                                     centos-sclo-rh                                21 k
 rh-mongodb34-runtime                                               x86_64                               3.0-14.el7                                      centos-sclo-rh                               1.2 M
 rh-mongodb34-yaml-cpp                                              x86_64                               0.5.3-1.el7                                     centos-sclo-rh                               181 k
 scl-utils                                                          x86_64                               20130529-18.el7_4                               base                                          24 k
 setools-libs                                                       x86_64                               3.3.8-2.el7                                     base                                         619 k
Updating for dependencies:
 audit                                                              x86_64                               2.8.1-3.el7                                     base                                         247 k
 audit-libs                                                         x86_64                               2.8.1-3.el7                                     base                                          99 k
 libselinux                                                         x86_64                               2.5-12.el7                                      base                                         162 k
 libselinux-python                                                  x86_64                               2.5-12.el7                                      base                                         235 k
 libselinux-utils                                                   x86_64                               2.5-12.el7                                      base                                         151 k
 libsemanage                                                        x86_64                               2.5-11.el7                                      base                                         150 k
 libsepol                                                           x86_64                               2.5-8.1.el7                                     base                                         297 k
 policycoreutils                                                    x86_64                               2.5-22.el7                                      base                                         867 k

Transaction Summary
============================================================================================================================================================================================================
Install  1 Package  (+27 Dependent packages)
~~~